src/index.ts

```diff
@@ -137,4 +145,44 @@ export default class BugzillaAPI {
 
     return this.searchBugs(searchParams);
   }
+
+  public async getComment(commentId: number) {
```

Comment:
Please add the return type to this and other methods.

Reply:
Done.